jetbrains-core/src/software/aws/toolkits/jetbrains/services/clouddebug/actions/InstrumentResourceAction.kt

```diff
@@ -12,6 +12,10 @@ import software.aws.toolkits.jetbrains.core.explorer.actions.SingleResourceNodeA
 import software.aws.toolkits.jetbrains.services.ecs.EcsServiceNode
 import software.aws.toolkits.jetbrains.services.ecs.EcsUtils
 import software.aws.toolkits.resources.message
+import software.aws.toolkits.telemetry.ClouddebugTelemetry
```

Comment:
each telemetry type is a separate class, which must be found, imported, and then called? 

Would it be  more ergonomic to have a single thing that's imported, which then has a bunch of members that can be called? Like:

```
import software.aws.toolkits.telemetry.Telemetry
...
Telemetry.clouddebug.instrument(...)
```

I don't understand why we need all of these classes/types.

Reply:
We can do it that way too, but IMO there is not much difference in ergonomics. You can see what classes are importable by typing `import software.aws.toolkits.telemetry` as well, so I don't have a strong opinion on which way is better.

Reply:
> You can see what classes are importable by typing `import software.aws.toolkits.telemetry` as well

If  `Telemetry` is where everything lives, then one always just types `Telemetry.<ctrl-space>` anywhere in any file, and gets a lives of options. 

With the current structure, one has to remember a bunch of different class names, or remember the `software.aws.toolkits.telemetry` namespace, or various other half-measures.

Reply:
I think if you are adding telemetry, you know what you are looking for already. I'd rather have the smaller files that are easier to grok